docs/docs_server_configuration_reference.md

```diff
@@ -248,3 +248,42 @@ Allows tuning settings related to in-game chat, such as message visibility radiu
   // ...
 }
 ```
+
+## npcEnabled
+
+Enables npc loading. By default is set to `false`.
+
+```json5
+{
+  // ...
+  "npcEnabled": false,
+  // ...
+}
+```
+
+## npcSettings
+
+Optional npcs configuration. May not be present or can be an empty object which means all npcs are allowed to be loaded, provided `"npcEnabled"` is set to `true`.
+`"NpcSettings"` consists of fields, each of which describes from what game file it is permitted to load an npc and additional restrictions of
+how they should be spawned: in interior or exterior. By default all the npcs are allowed (`"npcSettings": {}`).
+
+```json5
+{
+  // ...
+  "npcSettings": {
+    "Skyrim.esm": {
```

Comment:
please document what happens by default.

please add a way to specify default values of spawnInInterior/spawnInExterior for non-specified esp/esm files